omnipaxos/tests/config/test.toml

```diff
@@ -56,13 +56,10 @@ storage_type = { type = "Memory" }
 
 [batching_test]
 wait_timeout_ms = 3000
-num_thread = 8
-num_node = 3
-batch_siz = 31
-election_timeout_m = 50
-increment_dela = 2
-num_proposal = 310
-gc_id = 1000
+num_threads = 8
+num_nodes = 3
+batch_size = 1
```

Comment:
I think we used to use a prime number (31) because that would make checking entries getting decided in batches easier. But I guess if a server gets out of sync in the batches then this check wouldn't work. Anyway, we should use something larger than 1 here to test the batching at least.